core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -160,8 +161,8 @@ private Optimizely(@Nonnull ProjectConfig projectConfig,
         LogEvent impressionEvent =
                 eventBuilder.createImpressionEvent(projectConfig, experiment, variation, userId, attributes);
         logger.info("Activating user \"{}\" in experiment \"{}\".", userId, experiment.getKey());
-        logger.debug("Dispatching impression event to URL {} with params {}.", impressionEvent.getEndpointUrl(),
-                     impressionEvent.getRequestParams());
+        logger.debug("Dispatching impression event to URL {} with params {} and payload \"{}\".",
+                     impressionEvent.getEndpointUrl(), impressionEvent.getRequestParams(), impressionEvent.getBody());
```

Comment:
At any point, exclusively payload or params will be populated (get vs post requests). I'm including both because it's not worth slowing down the SDK by having a conditional to check whether we're sending a V1 or V2 event. Thus, the empty variable will be printed as "{}"